posthog/migrations/0204_remove_duplicate_plugin_configs.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.5 on 2021-11-30 08:23
+import django.utils.timezone
+from django.conf import settings
+from django.db import migrations, models
+
+from posthog.constants import AnalyticsDBMS
+
+
+def remove_duplicate_plugin_configs(apps, schema_editor):
+    PluginConfig = apps.get_model("posthog", "PluginConfig")
+    configs = PluginConfig.objects.raw(
+        """
+    select * from posthog_pluginconfig ou
```

Comment:
tiny nit that I'd format this statement better in order to make it nicer to read, like adding indentation for the subquery